src/DefaultBuilder/src/StaticWebAssetsReader.cs

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Xml.Linq;
+
+namespace Microsoft.AspNetCore
+{
+    internal static class StaticWebAssetsReader
```

Comment:
It's super weird that we use xml....

Reply:
Its friendlier on tooling because its available in netstandard. Otherwise It would have required to create a custom tool to bundle the json dependency so that the task would invoke a tool so that visual studio msbuild (full .net could call it) without having to load the json dependency into the build. This avoids unnecessary pain.